### DISCUSSION TITLE
After upgrade to 1.26.1 rPi3B is unresponsive and cannot connect to the Teslamate screens/sites

### DISCUSSION THREAD

**Discussion by M-ax-G** [teslamate-org/teslamate#2529] [Q&A] [answered]
Oh dear... what did I do...

Running rPi 3B with Buster Lite
Telsamate has been running since Oct 2021
I updated it once with:
docker-compose pull
docker-compose up -d
... as per doc, and how I did it last time.

The rPi uses all its memory and swap space.
I did a reboot, but the same unresponsiveness is seen.

Any hints appreciated.

![image](https://user-images.githubusercontent.com/21173730/157800731-cf621da3-3ac8-4620-a85d-ecfc90e8aca9.png)

**Comment by cwanja** [comment, marked as answer]
#2302 will give you a good overview. Adrian's comment specifically here https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011392890 

Couple of options:
1. Downgrade back to 1.25.0 
2. Backport the necessary files, learn more here https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011362938
3. Upgrade to Bullseye (or Bullseye Lite, if such exists). 

Hope it helps!

**Reply by M-ax-G** [reply]
Thank you kindly. That #2302 got me sorted...
I rebooted the rPi
connected via SSH
entered quickly: `docker ps -q | xargs docker stop` to stop all containers. Worked.
changed the `docker-compose.yml` by changin latest to 1.25.0
`docker-compose pull` and `docker-compose up -d`
And all seems back to normal.
Phew! Thank you; much appreciated. :)

**Reply by cwanja** [reply]
Just be aware that you will need to either upgrade your OS or backport the libraries _eventually_. TeslaMate core was upgraded, so it’s not like a future fix is coming in an update.

**Reply by M-ax-G** [reply]
Ahh... so, the way forward would be to upgrade the OS, and then update with 'latest'?!

**Reply by cwanja** [reply]
Correct - either upgrade to Bullseye or backport `libseccomp`. I am not sure about specifically Buster Lite and the backport functionality. But it is the fastest path to `latest` without an OS upgrade. https://github.com/adriankumpf/teslamate/issues/2302#issuecomment-1011362938

**Reply by M-ax-G** [reply]
OK, to complete this post...
I have upgraded to Bullseye, run docker pull and compose, and all is working.
Thanks for saving my day. :)
